paperweight-core/src/main/kotlin/taskcontainers/AllTasks.kt

```diff
@@ -57,7 +57,7 @@ open class AllTasks(
     val copyResources by tasks.registering<CopyResources> {
         inputJar.set(applyMergedAt.flatMap { it.outputJar })
         vanillaJar.set(extractFromBundler.flatMap { it.serverJar })
-        includes.set(listOf("/data/**", "/assets/**", "version.json", "yggdrasil_session_pubkey.der", "pack.mcmeta", "flightrecorder-config.jfc"))
+        excludes.set(extension.vanillaJarIncludes.map { it + listOf("META-INF/*") })
```

Comment:
Looking at this more I think
```suggestion
        excludes.convention(listOf("/META-INF/**", "**/*.class"))
```
would be better if we're gonna switch it from includes to excludes

Reply:
Okay :+1: Yea that should work out better especially for mutating the property.